src/ring_theory/ideal/operations.lean

```diff
@@ -21,8 +21,10 @@ namespace submodule
 variables {R : Type u} {M : Type v}
 variables [comm_ring R] [add_comm_group M] [module R M]
 
+open_locale pointwise
+
 instance has_scalar' : has_scalar (ideal R) (submodule R M) :=
-⟨λ I N, ⨆ r : I, N.map (r.1 • linear_map.id)⟩
+⟨λ I N, ⨆ r : I, (r : R) • N⟩
```

Comment:
Ideally we could just write `r • N`, but I ran out of steam when adding the subobject actions to `sub{monoid,group,ring}` and didn't make it as far as `submodule`.